src/commands/autocomplete/index.ts

```diff
@@ -67,32 +43,90 @@ export default class Index extends AutocompleteBase {
     ux.action.stop()
 
     if (!flags['refresh-cache']) {
-      const {bin} = this.config
-      const tabStr = shell === 'bash' ? '<TAB><TAB>' : '<TAB>'
+      this.printShellInstructions(shell)
+    }
+  }
+
+  private printShellInstructions(shell: string): void {
+    const setupEnvVar = this.getSetupEnvVar(shell)
+    const tabStr = shell === 'bash' ? '<TAB><TAB>' : '<TAB>'
+    const scriptCommand = `${this.config.bin} autocomplete${this.config.topicSeparator}script ${shell}`
+
+    let instructions = `
+Setup Instructions for ${this.config.bin.toUpperCase()} CLI Autocomplete ---
+==============================================
+`
+
+    switch (shell) {
+      case 'bash': {
+        instructions += `
+1) Run this command (starting with "printf") in your terminal window:
 
-      const instructions =
-        shell === 'powershell'
-          ? `New-Item -Type Directory -Path (Split-Path -Parent $PROFILE) -ErrorAction SilentlyContinue
-Add-Content -Path $PROFILE -Value (Invoke-Expression -Command "${bin} autocomplete${this.config.topicSeparator}script ${shell}"); .$PROFILE`
-          : `$ printf "eval $(${bin} autocomplete${this.config.topicSeparator}script ${shell})" >> ~/.${shell}rc; source ~/.${shell}rc`
+  ${chalk.cyan(`$ printf "eval $(${scriptCommand})" >> ~/.bashrc; source ~/.bashrc`)}
```

Comment:
@jshackell-sfdc comparing this with the `powershell` output where we don't include the `$` sign, what do you think if we just remove it? then we don't need to tell users to copy the stuff after `printf`

Reply:
@cristiand391:  yes, that's a good idea.  And I assume you mean to remove ALL the "$", right?  Including the ones  before "sf <TAB>"?   In all shell outputs?  That way it's consistent.

Reply:
correct, just left the `$` in the examples shown in `sf autocomplete --help` to be consistent with other commands.